### PR TITLE
Hugo profile pages

### PR DESCRIPTION
**Pull request by artur-rs**

### CODE REVIEW

File: blog/themes/3mdeb/layouts/authors/list.html [hunk 1]
```diff
@@ -1,58 +1,124 @@
+{{ partial "header.html" . }}
 {{ $pageAuthor := .Params.identifier }}
+{{ $paginator := .Paginate (where .Site.Pages "Section" "post").ByPublishDate.Reverse }}
```
Comment (lukasz-matysiewicz) [inline blog/themes/3mdeb/layouts/authors/list.html]: @artur-rs I tried use pagination code from index.html. At the moment it shows posts wrong order like it's in loop I think here is a problem: "Section" "post"

File: blog/themes/3mdeb/layouts/authors/terms.html [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+<div class="container">
+  <div class="section">
+    <div class="columns">
+      <div class="column is-9">
+        <div class="tile is-child box">
+          <div class="content">
+            {{ range .Data.Pages }}
+              <h2 class="author-title"><a href="{{ .Permalink }}">{{ .Params.name }}</a></h2>
+              <img class="author-image" src="{{ .Params.photo }}" alt=""/>
+              {{ $quantity := 0 }}
```
Comment (lukasz-matysiewicz) [inline blog/themes/3mdeb/layouts/authors/terms.html]: @artur-rs It count all Articles not per author

Comment (artur-rs) [reply blog/themes/3mdeb/layouts/authors/terms.html]: `{{ $pageAuthor := .Params.identifier}}` was set in the wrong place (not at the top, but in the `range` function). I'm making changes right now

File: blog/themes/3mdeb/layouts/authors/list.html (hunk 1 above)
Comment (artur-rs) [reply blog/themes/3mdeb/layouts/authors/list.html]: correct paginator: `{{ $paginator := .Paginate (where .Site.Pages "Params.author" $pageAuthor) }}`

Looped articles were caused by second `range` function

File: blog/themes/3mdeb/layouts/authors/terms.html [hunk 3]
```diff
@@ -1,6 +1,37 @@
-<h1>Authors</h1>
-<ul>
-{{ range .Data.Pages }}
-  <li><a href="{{ .Permalink }}">{{ .Params.name }}</a></li>
-{{ end }}
-</ul>
+{{ partial "header.html" . }}
+{{ $pageAuthor := .Params.identifier }}
```
[the remark this answers is not in the document]
Comment (artur-rs) [reply blog/themes/3mdeb/layouts/authors/terms.html]: my solution:
```
{{ else if eq .Type "authors" }}
    {{ if isset .Params "name" }}
        {{ .Params.name }}
    {{ else }}
        Authors
    {{ end }}
```

**Comment by artur-rs** [comment]
new mr: https://github.com/3mdeb/news-and-ideas/pull/140
closing this one

**Pull request closed without merging**